src/bp/core/config/config-loader.ts

```diff
@@ -129,18 +129,23 @@ export class ConfigProvider {
   }
 
   public async getModulesListConfig() {
-    const enabledByDefault = [
-      'analytics',
-      'basic-skills',
-      'builtin',
-      'channel-web',
-      'nlu',
-      'qna',
-      'extensions',
-      'code-editor',
-      'testing',
-      'examples'
-    ]
+    let enabledByDefault: string[]
+    if (process.env.ENABLED_BY_DEFAULT) {
+      enabledByDefault = process.env.ENABLED_BY_DEFAULT.replace(/\s/g, '').split(',')
+    } else {
+      enabledByDefault = [
+        'analytics',
+        'basic-skills',
+        'builtin',
+        'channel-web',
+        'nlu',
+        'qna',
+        'extensions',
+        'code-editor',
+        'testing',
+        'examples'
+      ]
+    }
```

Comment:
What do you think of adding a flag to enable ALL modules by default ? Instead of cherry picking modules, we could simply load all of them.

Ex: process.env.ENABLE_ALL_MODULES

This way, we will also be aware if someone breaks something in another module.

If you really only want to enable NLU-related modules, then I suggest adding something specific, ex:

```js
if(process.env.TEST_NLU){
  enabledByDefault = [...enabledByDefault, 'nlu-testing']
}
```

The current naming feels weird

Reply:
You're right!